docs/content/en/docs/yaml-crd-ref/_index.md

```diff
@@ -1,28 +1,22 @@
 ---
-title: CRD YAML Reference
-description: Reference pages for the YAML files that define the KLT CRDs
+title: Manifest CRD Reference
+description: Reference pages for the manifest YAML files that must be populated
```

Comment:
I don't think we need to emphasis that it's yaml so much. In the k8s world, everything is either yaml or json anyways (interchangebly)
This comment also applies to other places in this file